checkers/checker.go

```diff
@@ -8,13 +8,12 @@ import (
 	"path/filepath"
 
 	goAnalysis "globstar.dev/analysis"
```

Comment:
```suggestion
	"globstar.dev/analysis"
```

Reply:
Now that there's only the `analysis` package, and the `pkg/analysis` package is removed, you can import the `globstar.dev/analysis` package as-is, without renaming the import to `goAnalysis`. This comment applies to all the other places the name `goAnalysis` is being used.